### PR TITLE
[zh]Update content/zh/docs/concepts/storage/storage-capacity.md

### PR DESCRIPTION
**Pull request by zyy19981018**

### CODE REVIEW

File: content/zh/docs/concepts/storage/storage-capacity.md [hunk 1]
```diff
@@ -10,37 +10,47 @@ Storage capacity is limited and may vary depending on the node on
...
-{{< feature-state for_k8s_version="v1.21" state="beta" >}}
+{{< feature-state for_k8s_version="v1.24" state="stable" >}}
 
 This page describes how Kubernetes keeps track of storage capacity and
-how the scheduler uses that information to schedule Pods onto nodes
+how the scheduler uses that information to [schedule Pods](/docs/concepts/scheduling-eviction/) onto nodes
 that have access to enough storage capacity for the remaining missing
 volumes. Without storage capacity tracking, the scheduler may choose a
 node that doesn't have enough capacity to provision a volume and
 multiple scheduling retries will be needed.
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/storage-capacity.md:21]: You may want to translate the above paragraph and then the paragraph starting at line 24.
Do NOT translate large trunks of texts. It makes reviewing very difficult.

File: content/zh/docs/concepts/storage/storage-capacity.md [hunk 2]
```diff
@@ -10,37 +10,47 @@ Storage capacity is limited and may vary depending on the node on
...
+{{< feature-state for_k8s_version="v1.24" state="stable" >}}
 
 本页面描述了 Kubernetes 如何跟踪存储容量以及调度程序如何为了余下的尚未挂载的卷使用该信息将
-Pod 调度到能够访问到足够存储容量的节点上。
+[Pod 调度](/zh/docs/concepts/scheduling-eviction/)到能够访问到足够存储容量的节点上。
 如果没有跟踪存储容量，调度程序可能会选择一个没有足够容量来提供卷的节点，并且需要多次调度重试。
 
-{{< glossary_tooltip text="容器存储接口" term_id="csi" >}}（CSI）驱动程序支持跟踪存储容量，
-并且在安装 CSI 驱动程序时[需要启用](#enabling-storage-capacity-tracking)该功能。
+## {{% heading "prerequisites" %}}
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/storage-capacity.md:40]: 40 行后面加空行

File: content/zh/docs/concepts/storage/storage-capacity.md [hunk 3]
```diff
@@ -10,37 +10,47 @@ Storage capacity is limited and may vary depending on the node on
...
+Kubernetes v{{< skew currentVersion >}} 包括集群级 API 支持
+存储容量跟踪。 要使用它，你还必须使用 CSI 驱动程序
+支持容量跟踪。 查阅 CSI 驱动程序的文档
+你用于了解此支持是否可用，如果可用，如何使用它。 
+如果你没有运行 Kubernetes v{{< skew currentVersion >}}，
+请检查该版本 Kubernetes 的文档。
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/storage-capacity.md:46]: 参考[样式指南](https://kubernetes.io/zh/docs/contribute/localization_zh/)，请适当折行，不要在中文中间换行。

```suggestion
Kubernetes v{{< skew currentVersion >}} 包括集群级 API 支持存储容量跟踪。 
要使用它，你还必须使用支持容量跟踪的 CSI 驱动程序。 
查阅你所使用的 CSI 驱动程序的文档了解此支持是否可用，以及如果可用时如何使用它。 
如果你所运行的 Kubernetes 不是 v{{< skew currentVersion >}}，请检查对应版本 Kubernetes 的文档。
```

File: content/zh/docs/concepts/storage/storage-capacity.md [hunk 4]
```diff
@@ -52,7 +62,7 @@ There are two API extensions for this feature:
 ## API
 
 这个特性有两个 API 扩展接口：
-- CSIStorageCapacity 对象：这些对象由 CSI 驱动程序在安装驱动程序的命名空间中产生。
+- [CSIStorageCapacity](/zh/docs/reference/kubernetes-api/config-and-storage-resources/csi-storage-capacity-v1/)对象：这些对象由 CSI 驱动程序在安装驱动程序的命名空间中产生。
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/storage-capacity.md:65]: 这个不要加 `/zh` 前缀，API 文档尚未翻译。

Comment (tengqm) [reply content/zh/docs/concepts/storage/storage-capacity.md:65]:
```suggestion
- [CSIStorageCapacity](/docs/reference/kubernetes-api/config-and-storage-resources/csi-storage-capacity-v1/)
  对象：这些对象由 CSI 驱动程序在驱动程序所在的命名空间中生成。
```

**Comment by Sea-n** [comment]
This PR seems to be overlapped with `concepts-2`, next time it's better to assign yourself to [umbrella issues](https://github.com/kubernetes/website/issues?q=is%3Aissue+zh+Umbrella) to prevent conflict.

- #33590
- #33476

**Comment by Sea-n** [comment]
Unfortunately we have to close this PR. Still thanks for your contribution!
/close

**Pull request closed without merging**